test/integration/targets/freebsd_update/tasks/main.yml

```diff
@@ -0,0 +1,2 @@
+- include_tasks: tests.yml
+  when: (ansible_distribution == 'FreeBSD') and (ansible_distribution_version != '11.1')
```

Comment:
Why skip FreeBSD 11.1? It's EOL, but that's a scenario the module handles and should be tested for.

The test should be able to detect if the version being tested is EOL and act accordingly when verifying results.

Reply:
Yes, and no. Updating the base system to a newer release hasn't been implemented. Currently, the module only updates a release to its latest patch level.

Reply:
You might have noticed I skipped the `upgrade` command. To be fair, it's not an easy process, and I'm afraid it's not always automatable.

Reply:
No need to test `upgrade`, just test `update` even if the current version is EOL.